src/Core/src/Platform/iOS/MauiTextView.cs

```diff
@@ -176,18 +176,13 @@ void ShouldCenterVertically()
 
 		void UpdatePlaceholderFontSize(UIFont? value)
 		{
-			if (value != null)
+			if (_defaultPlaceholderSize is null)
 			{
-				if (_defaultPlaceholderSize == -1)
-				{
-					_defaultPlaceholderSize = _placeholderLabel.Font.PointSize;
-				}
-				_placeholderLabel.Font = _placeholderLabel.Font.WithSize(value.PointSize);
-			}
-			else if (_defaultPlaceholderSize != -1)
-			{
-				_placeholderLabel.Font = _placeholderLabel.Font.WithSize(_defaultPlaceholderSize);
+				_defaultPlaceholderSize = _placeholderLabel.Font.PointSize;
```

Comment:
Remove the condiiton and:

```suggestion
				_defaultPlaceholderSize ??= _placeholderLabel.Font.PointSize;
```

Reply:
good catch